.github/workflows/release-integration.yml

```diff
@@ -58,34 +58,36 @@ jobs:
       DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm" # Must be consistent with the matrix from the job above.
     steps:
       - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: actions/download-artifact@v2
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
 
       - name: Generate Docker image version from git tag
         run: |
           echo "${{ github.event.release.tag_name }}" | grep -E '^v[0-9.]*[0-9]$'
           DOCKER_IMAGE_TAG=$(echo "${{ github.event.release.tag_name }}" | sed 's/^v//')
           echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
-      - if: ${{ github.event.release.prerelease }}
+
+      - name: Change Docker image version if this run is a pre-release
```

Comment:
we could merge this step with the one releasing pre images, it is the only place we need the -pre suffix

Reply:
Solved in [#5fe777a](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/5fe777afae5b643f5965f056738b130343777573)